common/src/main/AndroidManifest.xml

```diff
@@ -13,7 +13,8 @@
         android:theme="@style/AppTheme">
         <activity
             android:name="com.microsoft.identity.common.internal.providers.oauth2.AuthorizationActivity"
-            android:configChanges="orientation|keyboardHidden|screenSize" />
+            android:configChanges="orientation|keyboardHidden|screenSize"
+            android:launchMode="singleTask"/>
```

Comment:
Would be good to include a comment about using singleTask in the actual activity.  Feels like we should make a note of this for future maintainers.